### ISSUE TITLE
Duplicated 'data.results' argument on the callback

### ISSUE THREAD

**Issue by josepramon** [edwardhotchkiss/mongoose-paginate#42]
On version 3.1.5, the data.results is passed twice on the callback.

The line:
```js
callback(null, data.results, Math.ceil(data.count / resultsPerPage) || 1, data.results, data.count);
```
should be:
```js
callback(null, data.results, Math.ceil(data.count / resultsPerPage) || 1, data.count);
```

**Comment by freezy** [comment]
Same issue here, all tests red ;)

**Comment by hustKiwi** [comment]
@niftylettuce

**Comment by CrackerakiUA** [comment]
+1, lets check source code and fix that, probably easy fix

**Comment by niftylettuce** [comment]
I'll look.

**Comment by niftylettuce** [comment]
Fixed and released v3.1.6 to NPM.  Thank you so much @josepramon @freezy

**Comment by freezy** [comment]
Everything back green. Cheers mate!
